python/ray/util/client/worker.py

```diff
@@ -52,6 +54,19 @@
 # operations.
 MAX_BLOCKING_OPERATION_TIME_S = 2
 
+# If the total size (bytes) of all outbound messages to schedule tasks since
+# the connection began exceeds this value, a warning should be raised
+MESSAGE_SIZE_THRESHOLD = 10 * 2**20  # 10 MB
+
+# If the number of tasks scheduled on the client side since the connection
+# began exceeds this value, a warning should be raised
+TASK_WARNING_THRESHOLD = 1000
+
+# Link to the Ray Design Pattern doc to use in the task overhead warning
+# message
+DESIGN_PATTERN_DOC_LINK = \
+    "https://docs.google.com/document/d/167rnnDFIVRhHhK4mznEIemOtj63IOhtIPvSYaPgI4Fg/" # noqa E501
```

Comment:
can you please use this link (which goes directly to the subsection):
https://docs.google.com/document/d/167rnnDFIVRhHhK4mznEIemOtj63IOhtIPvSYaPgI4Fg/edit#heading=h.f7ins22n6nyl